Common/DataModel/TrackSelectionTables.h

```diff
@@ -19,21 +19,79 @@ namespace o2::aod
 namespace track
 {
 // Columns to store the DCA to the primary vertex
-DECLARE_SOA_COLUMN(DcaXY, dcaXY, float); //!
-DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);   //!
+DECLARE_SOA_COLUMN(DcaXY, dcaXY, float); //! Impact parameter in XY of the track to the primary vertex
+DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);   //! Impact parameter in Z of the track to the primary vertex
+
+struct TrackSelectionFlags {
+ public:
+  typedef uint16_t flagtype;
+  static constexpr flagtype kTrackType = 1 << 0;
+  static constexpr flagtype kPtRange = 1 << 1;
+  static constexpr flagtype kEtaRange = 1 << 2;
+  static constexpr flagtype kTPCNCls = 1 << 3;
+  static constexpr flagtype kTPCCrossedRows = 1 << 4;
+  static constexpr flagtype kTPCCrossedRowsOverNCls = 1 << 5;
+  static constexpr flagtype kTPCChi2NDF = 1 << 6;
+  static constexpr flagtype kTPCRefit = 1 << 7;
+  static constexpr flagtype kITSNCls = 1 << 8;
+  static constexpr flagtype kITSChi2NDF = 1 << 9;
+  static constexpr flagtype kITSRefit = 1 << 10;
+  static constexpr flagtype kITSHits = 1 << 11;
+  static constexpr flagtype kGoldenChi2 = 1 << 12;
+  static constexpr flagtype kDCAxy = 1 << 13;
+  static constexpr flagtype kDCAz = 1 << 14;
+  static constexpr flagtype kGlobalTrack = kTrackType | kPtRange | kEtaRange | kTPCNCls | kTPCCrossedRows | kTPCCrossedRowsOverNCls | kTPCChi2NDF | kTPCRefit | kITSNCls | kITSChi2NDF | kITSRefit | kITSHits | kGoldenChi2 | kDCAxy | kDCAz;
```

Comment:
Will these be the same for Run 2 and Run 3?

Reply:
yes, the change is in the track selection class

Reply:
I mean, will we not have different cuts for Run 3?

Reply:
Yes, but the bits of the bitmask will be the same. Then the bitmask will be filled differently for Run2 and Run3